includes/class-wc-structured-data.php

```diff
@@ -210,7 +210,7 @@ public function generate_product_data( $product = null ) {
 		}
 
 		$markup['image']       = wp_get_attachment_url( $product->get_image_id() );
-		$markup['description'] = wpautop( do_shortcode( $product->get_short_description() ? $product->get_short_description() : $product->get_description() ) );
+		$markup['description'] = wp_filter_nohtml_kses( do_shortcode( $product->get_short_description() ? $product->get_short_description() : $product->get_description() ) );
```

Comment:
Why KSES instead of something like `wp_strip_all_tags`?

Reply:
I was under the impression that kses was a bit more robust/thorough.

Looking into it its prob best to use wp_strip_all_tags though, as its less heavy on resources I do believe?

Reply:
Hi Mike, excuse my ignorance here, I was going to try and change wp_filter_nohtml_kses to wp_strip_all_tags - can I still do this on this pull request?

Reply:
@noelmcg It's ok, I handled it when I merged into master.

Reply:
@mikejolley 👍 cheers